bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/view/partner/PartnerView.scala

```diff
@@ -887,16 +887,6 @@ class PartnerView extends BorderPanel with BoundView[Proposal] {view =>
         // Enabled?
         enabled = canEdit
 
-        // Hide CFHT for queue proposals
-        m.map {
-          case _:QueueProposalClass     =>
-            new DefaultComboBoxModel(PartnerType.values.filterNot(_ == ExchangeCFHT).toArray)
-          case _:ClassicalProposalClass =>
-            new DefaultComboBoxModel(PartnerType.values.toArray)
-          case _                        =>
-            this.peer.getModel
-        }.map(this.peer.setModel)
```

Comment:
Use `foreach` if you're going to side-effect; this ensures that it happens now and exactly once. `map` doesn't guarantee either.